docs/release_instructions.md

```diff
@@ -290,11 +290,11 @@ into other runtime projects to build their images.
 
 ### Publishing to npm
 
-The openwhisk-client-js and openwhisk-composer project release npm
-packages built from each source release.  The Release Manager should
-build and publish these packages manually using the openwhisk-bot
-credentials found in the npmjs.txt file in the accounts subdir of the
-PMC private svn.
+The openwhisk-client-js, openwhisk-composer, and openwhisk-wskdebug
+project release npm packages built from each source release.  The
+Release Manager should build and publish these packages manually using
+the openwhisk-bot credentials found in the npmjs.txt file in the
+accounts subdir of the PMC private svn.
```

Comment:
should we provide a link to it here for convenience or leave it out for _security_?

Reply:
I'd be inclined to leave it out.  Hopefully PMC members know where to find this (and will ask on @private if they don't).